exercises/practice/sgf-parsing/src/test/java/SgfParsingTest.java

```diff
@@ -12,27 +12,27 @@ public class SgfParsingTest {
     @Test
     public void emptyInput() {
         String input = "";
-        assertThrows("tree missing",
-            SgfParsingException.class,
-            () -> new SgfParsing().parse(input));
+        assertThatExceptionOfType(SgfParsingException.class)
+                .isThrownBy(() -> new SgfParsing().parse(input))
+                .withMessage("tree missing");
```

Comment:
This `.withMessage()` is incorrect: the first argument to JUnit's `assertThrows()` doesn't assert the exception message but is used as the message printed when the assertion fails. 

In fact, the original test doesn't perform an assertion of the exception message, only on its type, so there is no need to call `.withMessage()` in any of the tests in this file.